dojo/tools/mend_sca_platform_api3/parser.py

```diff
@@ -0,0 +1,169 @@
+import hashlib
+import json
+import logging
+
+from dojo.models import Finding
+
+__author__ = "testaccount90009 aka SH"
+
+logger = logging.getLogger(__name__)
+
+
+class MendPlatformApi3Parser:
```

Comment:
From my point of view it would be better to harden the already existing parser to be able to process this new file structure.

Reply:
There are several kinds of Mend scans and APIs that are different.  Platform SAST, IaC, SCA are different than previous 1.0 or 2.0 API structured Findings.  I think this should be a separate parser, the same way that AWS has multiple parsers depending on the data source.  To write one parser that would accomplish all 1.0, 2.0 SCA + Platform SAST, SCA, IaC -- would be far more intricate than just adding a 2nd parser and differentiating Legacy (1.0 + 2.0) vs new 'Platform' and 3.0 API structured Findings.

Reply:
@manuel-sommer as of the latest commits, I removed all references to an additional parser and instead updated the original Mend parser areas.  Thank you for the guidance and suggestions!  Assuming unit tests pass, I can pull these changes into my Dev stack and test an import and see if DefectDojo accepts it.